docs/en/advanced_tutorials/registry.md

```diff
@@ -272,6 +272,31 @@ output = model(input)
 print(output)
 ```
 
+### How does the parent node know about child registry?
+
+When working in our `MMAlpha` it might be necessary to use the `Runner` class defined in MMEngine. This class is in charge of building most of the objects. If this objects are added to the child registry (`MMAlpha`), how is  `MMEngine` able to find them? It cannot, `MMEngine` needs to switch to the Registry from `MMEngine` to `MMAlpha` according to the `default_scope` argument when building the runner (commonly defined in `default_runtime.py` of downstream repositories) for searching the target class.
+
+We can also init the scope accordingly, see example below:
+
+```python
+from mmalpha.registry import MODELS
+from mmengine.registry import init_default_scope
+
+@MODELS.register_module()
+class LogSoftmax(nn.Module):
+    def __init__(self, dim=None):
+        super().__init__()
+
+    def forward(self, x):
+        print('call LogSoftmax.forward')
+        return x
+
+assert "LogSoftmax" in MODELS # This fails since registry is looking only in parent class
```

Comment:
Hi, I found that this assertion will not False in v0.8.0. I've also tried an old version such as v0.3.0, this assertion will still succeed.  I think a more proper example here is to build the instance by `MODLES.build(dict(type='ToyModel'))`, and it will fail if `init_default_scope('mmalpha')` have not been called before.

Reply:
I tried it too and you are right, that was not working as expected. I updated the example, please have a look and let me know what you think :)